src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination:React.FC<Props> = (props) => {
+  const {
+    total,
+    perPage,
+    currentPage,
+    onPageChange,
+  } = props;
+  const pagesCount = Math.ceil(total / perPage);
+  const pages = Array.from(Array(pagesCount).keys());
+
+  return (
+    <>
+      <ul className="pagination">
```

Comment:
```suggestion
    <ul className="pagination">
```
Do you really need react fragment here? You already have a container